fth/file.fth

```diff
@@ -85,4 +87,19 @@ create (LINE-TERMINATOR) \n c,
     THEN
 ;
 
+: RENAME-FILE ( c-addr1 u1 c-addr2 u2 -- ior )
+    { a1 u1 a2 u2 | new }
+    \ Convert the file-names to C-strings by copying them after HERE
+    \ with trailing zeros added.
+    a1 here u1 move
```

Comment:
Please define a word that copies a string and adds a NUL. 
( c-addr1 u1 c-addr2 -- )